packages/base/src/3dview/mainview.tsx

```diff
@@ -622,6 +624,17 @@ export class MainView extends React.Component<IProps, IStates> {
           }
         });
         this._meshGroup?.add(meshGroup);
+        // Apply the objColor to the mainMesh
+        if (objColor && mainMesh?.material?.color) {
+          const selectedMesh = this._meshGroup?.getObjectByName(
+            mainMesh.name
+          ) as BasicMesh;
+          if (selectedMesh && selectedMesh.material?.color) {
+            const newColor = new THREE.Color(objColor);
+            selectedMesh.material.color = newColor;
+            selectedMesh.userData.originalColor = newColor.clone();
+          }
+        }
```

Comment:
This is overwriting the initialization color of selected meshes (done line 622 of your file). 

You probably need to put this code here in and `else` fallback for the `if` condition line 620.

This is what causes the visual regression.